specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/dataVersion.json

```diff
@@ -0,0 +1,387 @@
+    },
+    "DataVersionVersionsParameter": {
+      "in": "path",
+      "name": "version",
+      "type": "string",
+      "description": "Data Version Version identifier.",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "definitions": {
+    "DataVersionResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The resource URL of the entity (not URL encoded).",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the resource entity.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource provider and type.",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/DataVersion"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "additionalProperties": false,
+      "description": "Azure Resource Manager resource Envelope",
+      "x-ms-azure-resource": true
+    },
+    "DataVersionResourcePaginatedResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataVersionResource"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "DataVersion": {
+      "description": "Details of an Data Version",
+      "type": "object",
+      "properties": {
+        "datasetType": {
+          "$ref": "#/definitions/DatasetType"
+        },
+        "description": {
+          "description": "The asset description text.",
+          "type": "string"
+        },
+        "datastoreId": {
+          "description": "The asset datastoreId",
+          "type": "string"
+        },
+        "tags": {
+          "description": "The asset tag dictionary.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "The asset property dictionary.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "assetPaths": {
+          "description": "Details of asset paths in the datastore.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssetPath"
+          }
+        },
+        "linkedResourceIds": {
+          "description": "Associated ARM resources. Key is the link type, value is a list of ARM IDs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "DatasetType": {
```

Comment:
You are encouraged to use x-ms-enum with ModelAsString: true, this will prevent new values from causing breaking changes